admin_frontend/templates/components/invite.html

```diff
@@ -0,0 +1,43 @@
+<div id="create-user">
+  <h4>Please enter the following email invite a new user</h4>
+  <table>
+    <tr>
+      <td>Email:</td>
+      <td>
+        <input
+          class="input"
+          id="invitedUserEmail"
+          placeholder="myfriend@example.com"
+        />
+      </td>
+    </tr>
+    <tr>
+      <td></td>
+      <td style="text-align: right">
+        <button class="button cyan" id="inviteBtn">Invite</button>
+      </td>
+    </tr>
+  </table>
+
+  <script>
+    document.getElementById("inviteBtn").addEventListener("click", function () {
```

Comment:
If we choose not to use any web frameworks (like React) here, we may need to refactor the HTML

Reply:
Most of the HTML are components and can be reused when necessary to keep the code dry. Is there a specific part you are referring to?

Reply:
No, it's just a concern